composer.json

```diff
@@ -126,6 +126,7 @@
         "drupal/search_api_attachments": "9.0.0",
         "drupal/search_api_solr": "4.2.7",
         "drupal/seckit": "2.0.0",
+        "drupal/securitytxt": "^1.2",
```

Comment:
we have to specify the exactly version to 1.2

Reply:
Done